Lib/fontTools/feaLib/builder_test.py

```diff
@@ -361,6 +363,92 @@ def test_lookup_inside_feature_aalt(self):
+                    res = "pos base {}".format(self.base.asFea())
+                    for a, m in self.marks:
+                        res += " {} mark @{}".format(a.asFea(), m.name)
+                    res += ";"
+                return res
+
+        class testAst(object) :
+            MarkBasePosStatement = ast_MarkBasePosStatement
+            def __getattr__(self, name):
+                return getattr(ast, name)
+
+        class testParser(Parser):
+            def parse_position_base_(self, enumerated, vertical):
+                location = self.cur_token_location_
+                self.expect_keyword_("base")
+                if enumerated:
+                    raise FeatureLibError(
+                        '"enumerate" is not allowed with '
+                        'mark-to-base attachment positioning',
+                        location)
+                base = self.parse_glyphclass_(accept_glyphname=True)
+                if self.next_token_ == "<" :
+                    marks = self.parse_anchor_marks_()
+                else:
+                    marks = []
+                    while self.next_token_ == "mark":
+                        self.expect_keyword_("mark")
+                        m = self.expect_markClass_reference_()
+                        marks.append(m)
+                self.expect_symbol_(";")
+                return self.ast.MarkBasePosStatement(location, base, marks)
+
+            def parseBaseClass(self):
+                if not hasattr(self.doc_, 'baseClasses'):
+                    self.doc_.baseClasses = {}
+                location = self.cur_token_location_
+                glyphs = self.parse_glyphclass_(accept_glyphname=True)
+                anchor = self.parse_anchor_()
+                name = self.expect_class_name_()
+                self.expect_symbol_(";")
+                baseClass = self.doc_.baseClasses.get(name)
+                if baseClass is None :
+                    baseClass = ast_BaseClass(name)
+                    self.doc_.baseClasses[name] = baseClass
+                    self.glyphclasses_.define(name, baseClass)
+                bcdef = ast_BaseClassDefinition(location, baseClass, anchor, glyphs)
+                baseClass.addDefinition(bcdef)
+                return bcdef
+
+            extensions = {
+                'baseClass' : lambda s : s.parseBaseClass()
+            }
+            ast = testAst()
+
+        f = self.getpath("baseClass.feax")
+        p = testParser(f)
+        #import pdb; pdb.set_trace()
+        doc = p.parse()
+        tlines = self.normal_fea(doc.asFea().split("\n"))
+        with open(self.getpath("baseClass.fea"), "r", encoding="utf-8") as ofile:
+            olines = self.normal_fea(ofile.readlines())
+        if olines != tlines:
+            for line in difflib.unified_diff(olines, tlines):
+                sys.stderr.write(line)
+            self.fail("Fea2Fea output is different from expected:\n{}".format("\n".join(tlines)))
```

Comment:
maybe you could wrap this code in an `expect_fea` helper method, like the similar `expect_ttx` one.